### PR TITLE
Add freight endpoints + rename ton miles

### PR DESCRIPTION
**Pull request by valeriiashmyhlo**
- Rename 'ton miles' -> 'tonne-miles' in endpoint, titles, comments, (web: VorTECHsa/web#2337 and app-core: VorTECHsa/app-core#541 already merged)
- Add Fleet Utilisation endpoints
- add vessel availability endpoints

Resolves: VorTECHsa/app-issues#1030

### CODE REVIEW

File: pydocmd.yml [hunk 1]
```diff
@@ -60,7 +82,17 @@ pages:
       Products: endpoints/products.md
       Vessels: endpoints/vessels.md
       EIA Forecasts: endpoints/eia_forecasts.md
-      Ton Miles Breakdown: endpoints/ton_miles_breakdown.md
+      Tonne-miles Breakdown: endpoints/tonne_miles_breakdown.md
+      Vessel Availability: endpoints/availability_search.md
+      Vessel Availability Time Series: endpoints/availability_timeseries.md
```
Comment (andreagerino) [inline pydocmd.yml:87]: Can you please uniform all usage of "Time Series": either "Time Series" or "Timeseries" in all documents.

File: vortexasdk/api/entity_flattening.py [hunk 2]
```diff
@@ -43,6 +54,9 @@ def convert_vessel_movement_to_flat_dict(vm: Dict, cols="all") -> Dict:
         return {k: v for k, v in formatted.items() if k in cols}
 
 
+def _group_vessel_availability_attributes_by_layer(va: Dict) -> Dict:
+    return va
```
Comment (amhchiu) [inline vortexasdk/api/entity_flattening.py:58]: Does this do anything?

File: vortexasdk/operations.py [hunk 3]
```diff
@@ -66,7 +66,7 @@ def __init__(self, resource):
         self._resource = resource
 
     def search(
-        self, exact_term_match: bool = None, **api_params
+        self, exact_term_match: bool = None, query_type: str = None, **api_params
```
Comment (amhchiu) [inline vortexasdk/operations.py:69]: worth adding `query_type` under `Arguments` description?

File: vortexasdk/api/breakdown_item.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+@dataclass(frozen=True)
+class BreakdownItem(FromDictMixin):
+    """
+    Generic container class holding a _key_, _value_ pair, and a _count_ and breakdown of records contributing to the given value.
+
+    For example, this class could hold the average speed of vessels (value) on 2019-01-01 (key), the number of vessels contributing to the
+    this average (count) and additional information about the aggregation (breakdown).
+    """
+
+    key: ISODate
```
Comment (andreagerino) [inline vortexasdk/api/breakdown_item.py:17]: What about making key `string ` and the breakdown field optional?
This way we would remove the need to have a separate geo_breakdown type

Comment (andreagerino) [reply vortexasdk/api/breakdown_item.py:17]: P.s. this if neither breakdown or geo_breakdown support paging

File: vortexasdk/api/breakdown_item.py [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
 
 @dataclass(frozen=True)
 class BreakdownItem(FromDictMixin):
     """
-    Generic container class holding a _key_, _value_ pair, and a _count_ and breakdown of records contributing to the given value.
+    Generic container class holding a `key <> value` pair, a `count`, and optionally a `label` and a `breakdown` of records contributing to the given value.
 
-    For example, this class could hold the average speed of vessels (value) on 2019-01-01 (key), the number of vessels contributing to the
-    this average (count) and additional information about the aggregation (breakdown).
+    For example, this class could hold the average speed of vessels (`value`) on 2019-01-01 (`key), the number of vessels contributing to the
```
Comment (amhchiu) [inline vortexasdk/api/breakdown_item.py:12]: 2019-01-01 (`key`) <- missing the escape character

File: vortexasdk/endpoints/fleet_utilisation_destination_breakdown.py [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+            exclude_vessel_ice_class: An attribute ID, or list of attribute IDs to exclude.
+
+            exclude_vessel_propulsion: An attribute ID, or list of attribute IDs to exclude.
+
+
+        # Returns
+        `ReferenceBreakdownResult`
+
+        # Example
+        _Top 5 countries by number of unique vessels by origin country breakdown, in the last quarter._
```
Comment (amhchiu) [inline vortexasdk/endpoints/fleet_utilisation_destination_breakdown.py:143]: Should this say 'by destination country'?

**Review by amhchiu — APPROVED** [review]

**Pull request merged**